.eslintrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
   extends: "@vue-storefront/eslint-config-integrations",
+  rules: {
+    "class-methods-use-this": "off",
```

Comment:
This is good enough, but I think that we could add it to the `@vue-storefront/eslint-config-integrations` package directly